ts/types/wallet.ts

```diff
@@ -0,0 +1,28 @@
+export type PaymentMethod = {
+  id: number;
+  type: string;
+};
+
+export type Operation = {
```

Comment:
I'd rename this "WalletOperation" (to avoid confusion when used in other parts of the code)

Reply:
I actually wanted to rename this based on the general consensus: do you happen to know whether the commonly agreed upon term is operation, transaction, payment, or something else entirely?

Reply:
Is there a related concept in pagopa APIs ? (what's its name there ? maybe we can reuse it)

Reply:
The same concept is referred to as a transaction in PagoPA iirc, will update this to WalletTransaction